src/utils/timeoutValues.js

```diff
@@ -0,0 +1,10 @@
+export const ONE_MINUTE_IN_MS = 60000;
+
+/**
+ * We change the values according to the chain_id
+ */
+export const TIMEOUT_MAP = {
+  33: ONE_MINUTE_IN_MS * 5,
```

Comment:
This should be 33 for regtest, 31 and 30 for testnet and mainnet

Reply:
My bad, forgot about 30

Reply:
Added it